### PR TITLE
test(plugin-lighthouse): add lighthouse CI flags to configs

### LINKED ISSUE #657: Lighthouse tests are flaky
### What happened?

In general the lighthouse setup in the CI is not super robust.

### What would you expect to happen?

We could include the flags from chrome launcher to see if the CI gets more stable:
- https://github.com/GoogleChrome/chrome-launcher/blob/main/docs/chrome-flags-for-tools.md

### What steps did you take?

I read the source to figure it out

### Code PushUp package version

_No response_

### What operation system are you on?

Linux

### Node version

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by BioPhoton**
When using Chrome it helps to have a set of flags for more stable runs in a tooling environment like thee CI.
The `chrome-launcher` package provides a set of flags dedicated to tooling that they also documented very well.

This PR includes:
- docs
- uses tooling flags in both configs

closes #657

### CODE REVIEW

File: code-pushup.config.ts [hunk 1]
```diff
@@ -77,7 +78,9 @@ const config: CoreConfig = {
       type: 'module',
     }),
 
-    await lighthousePlugin('https://codepushup.dev/'),
+    await lighthousePlugin('https://codepushup.dev/', {
+      chromeFlags: DEFAULT_FLAGS.concat(['--headless']),
```
Comment (matejchalk) [inline code-pushup.config.ts:82]: Should we set these Chrome flags by default and make `chrome-launcher` a dependency of `@code-pushup/lighthouse-plugin`? :thinking:

File: packages/plugin-lighthouse/README.md [hunk 2]
```diff
@@ -147,6 +147,19 @@ For a complete list the [official documentation of CLI flags](https://github.com
 > - `list-trace-categories` - Prints a list of all required trace categories and exits.
 > - `view` - Open HTML report in your browser
 
+## Chrome Flags for Tooling
+
+We recommend using Chrome flags for more stable runs in a tooling environment.
+The `chrome-launcher` package provides a set of flags dedicated to tooling that they also documented very well.
```
Comment (matejchalk) [inline packages/plugin-lighthouse/README.md:153]: Linking to those docs would be convenient :slightly_smiling_face: 

```suggestion
The [`chrome-launcher`](https://www.npmjs.com/package/chrome-launcher) package provides a set of flags dedicated to tooling that they also documented very well.
```

File: code-pushup.config.ts (hunk 1 above)
Comment (BioPhoton) [reply code-pushup.config.ts:82]: Thought about it and decided against listing it as dependency. 
But is should be indirectly here anyway... https://github.com/GoogleChrome/lighthouse/blob/369979f498bd6560127e10476edffb264d4fa3b9/cli/run.js#L13

File: packages/plugin-lighthouse/README.md [hunk 3]
```diff
@@ -147,6 +147,19 @@ For a complete list the [official documentation of CLI flags](https://github.com
 > - `list-trace-categories` - Prints a list of all required trace categories and exits.
 > - `view` - Open HTML report in your browser
 
+## Chrome Flags for Tooling
+
+We recommend using Chrome flags for more stable runs in a tooling environment.
+The [`chrome-launcher`](https://www.npmjs.com/package/chrome-launcher) package provides a set of flags dedicated to tooling that they also documented very well.
+``
```
Comment (matejchalk) [inline packages/plugin-lighthouse/README.md:154]:
```suggestion
```

File: packages/plugin-lighthouse/README.md [hunk 4]
```diff
@@ -147,6 +147,19 @@ For a complete list the [official documentation of CLI flags](https://github.com
...
+
+> ```ts
+> // code-pushup.config.ts
+> import { DEFAULT_FLAGS } from 'chrome-launcher/dist/flags.js';
+> ...
+> lighthousePlugin('https://example.com', { output: 'json', chromeFlags: DEFAULT_FLAGS});
```
Comment (matejchalk) [inline packages/plugin-lighthouse/README.md:160]:
```suggestion
> lighthousePlugin('https://example.com', { output: 'json', chromeFlags: DEFAULT_FLAGS });
```

File: code-pushup.config.ts (hunk 1 above)
Comment (matejchalk) [reply code-pushup.config.ts:82]: What's the argument for not making these flags the default? Based on your description of "more stables runs", it sounds like something I would want on every project.

Comment (BioPhoton) [reply code-pushup.config.ts:82]: I also believe it, I am just a bit unsure and wanted to have it in use for a while. 
I could surely move it into the plugin.

**Review by matejchalk — APPROVED** [review]

**Pull request merged**